ethclient/ethclient.go

```diff
@@ -664,6 +664,15 @@ func toCallArg(msg ethereum.CallMsg) interface{} {
 	if msg.GasTipCap != nil {
 		arg["maxPriorityFeePerGas"] = (*hexutil.Big)(msg.GasTipCap)
 	}
+	if msg.AccessList != nil {
+		arg["accessList"] = msg.AccessList
+	}
+	if msg.BlobGasFeeCap != nil {
```

Comment:
What is the purpose of this parameter?

Reply:
It's used [here](https://github.com/ethereum-optimism/op-geth/blob/84b52dfd2626f5e19da6baa2691d81f4c2299eb5/core/state_transition.go#L378), so that a blob tx will not be included if the specified blob gas fee cap is smaller than `BlobBaseFee` of current block.

Reply:
Here's more context about this change:https://github.com/ethereum-optimism/op-geth/pull/299